lib/anagrams.rb

```diff
@@ -1,2 +1,13 @@
 def anagrams?(str1, str2)
-end
\ No newline at end of file
+    array_1 = str1.split("")
+    array_2 = str2.split("")
+    array_3 = array_1.select { |el| !array_2.include?(el) }
+        if array_1.length == array_2.length &&
```

Comment:
Oh, dear! LOL! I don’t know how or why I did that. My brain has been like jello lately. That is definitely redundant. Do you want me to go ahead and fix that and resubmit it?


> On Jan 12, 2016, at 5:57 PM, Sumeet Jain <notifications@github.com> wrote:
> 
> In lib/anagrams.rb <https://github.com/omahacodeschool/ruby-toy__check-if-anagrams/pull/3#discussion_r49533867>:
> 
> > \ No newline at end of file
> > +    array_1 = str1.split("")
> > +    array_2 = str2.split("")
> > +    array_3 = array_1.select { |el| !array_2.include?(el) }
> > +        if array_1.length == array_2.length &&
> This conditional and its branches should be indented one to the left.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/omahacodeschool/ruby-toy__check-if-anagrams/pull/3/files#r49533867>.
>